framework/test/jest/unit/specs/modules/chain/bft/utils.spec.js

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright © 2018 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ */
+
+'use strict';
+
+const { SchemaValidationError } = require('../../../../../../../src/errors');
+const {
+	BlockHeader: blockHeaderFixture,
+} = require('../../../../../../mocha/fixtures/blocks');
+
+const {
+	validateBlockHeader,
+} = require('../../../../../../../src/modules/chain/bft/utils');
+
+describe('utils', () => {
+	describe('validateBlockHeader', () => {
+		it('should be ok for valid headers', async () => {
+			const header = blockHeaderFixture();
```

Comment:
please use // Arrange & Act & Assert comments in the tests for better readability.

Reply:
I felt code itself is very sell explanatory and not needed such comments for few lines of blocks.